siliconcompiler/schema/schema_cfg.py

```diff
@@ -2988,6 +2991,53 @@ def schema_option(cfg):
     return cfg
 
 
+############################################
+# Dependency information
+############################################
+def schema_dependency(cfg):
+
+    dependency = 'default'
+
+    scparam(cfg, ['dependency', dependency, 'path'],
+            sctype='str',
+            scope='global',
+            shorthelp="Dependency source path",
+            switch="-dependency_path 'dependency <str>'",
+            example=[
+                "cli: -dependency_path "
+                "'freepdk45_data ssh://git@github.com/siliconcompiler/freepdk45/'",
+                "api: chip.set('dependency', "
+                "'freepdk45_data', 'path', 'ssh://git@github.com/siliconcompiler/freepdk45/')"],
+            schelp="""
+            Dependency source path
+            """)
+
+    scparam(cfg, ['dependency', dependency, 'commitid'],
+            sctype='str',
+            scope='global',
+            shorthelp="Dependency source commitid",
+            switch="-dependency_commitid 'dependency <str>'",
+            example=[
+                "cli: -dependency_commitid 'freepdk45_data 07ec4aa'",
+                "api: chip.set('dependency', 'freepdk45_data', 'commitid', '07ec4aa')"],
+            schelp="""
+            Dependency source commitid""")
```

Comment:
maybe `version` is a bit more generic? I'm not sure about that, just a suggestion.

Reply:
ref is probably good